mcp/src/main/java/io/modelcontextprotocol/spec/McpSchema.java

```diff
@@ -134,6 +134,18 @@ public static final class ErrorCodes {
 
 	}
 
+	// ---------------------------
+	// JSON-RPC Protocol Headers
+	// ---------------------------
+	/**
+	 * Standard header name used in MCP JSON-RPC request and responses.
+	 */
+	public static final class Headers {
```

Comment:
I feel that this belongs around something like `McpHttpTransport` interface rather than in the `McpSpec`, which is more about the JSON-RPC layer.

Reply:
The primary purpose here is to provide users with the MCP standard header name information. The `MCP-Session-Id `does belong to the transport category. Of course, I have omitted the `last-event-id `here. At present, I think defining these header names as part of the MCPSchema is the most reasonable approach because I haven't thought of any other more reasonable alternatives. These header names are indeed mentioned in the Specification. Do you have any better suggestions? @chemicL